test/840_weave_kube_3_test.sh

```diff
@@ -2,6 +2,8 @@
 
 . "$(dirname "$0")/config.sh"
 
+cd 
```

Comment:
Definitely a mistake on my end -- removing it now.
Sorry about that and well spotted.